packages/core/src/modules/mdoc/MdocDeviceResponse.ts

```diff
@@ -26,7 +27,41 @@ import { getMdocContext } from './MdocContext'
 import { MdocError } from './MdocError'
 
 export class MdocDeviceResponse {
-  public constructor() {}
+  private constructor(public base64Url: string, public documents: Mdoc[]) {}
+
+  public static isBase64DeviceResponse(base64Url: string) {
+    try {
+      const parsed = parseDeviceResponse(TypedArrayEncoder.fromBase64(base64Url))
+      if (parsed.status === MDocStatus.OK) return true
+      return false
+    } catch (error) {
+      // no-op
+    }
+
+    return false
```

Comment:
```suggestion
    try {
      const parsed = parseDeviceResponse(TypedArrayEncoder.fromBase64(base64Url))
      return parsed.status === MDocStatus.OK
    } catch (error) {
      return false
    }
```

Reply:
Also seems quite expensive. That we try to parse. But maybe the best way. Are we doing in places:
```
if (isBase64DeviceReponse(data)) {
   parseBase64DeviceResponse()
}
```

Because in that case a `tryParseBase64DeviceResponse` that returns null on error might be better?